docs/wiki/framework/atragmx.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: wiki
+title: ATragMX Framework
+description: Explains how to use the ATragMX framework.
```

Comment:
Could the description be a little bit improved?

Reply:
I'll try.